usage/scripts/change-version.sh

```diff
@@ -63,7 +63,7 @@ FILES_COUNT=`echo $FILES | wc | awk '{print $2}'`
 if [ ${FILES_COUNT} -ne 0 ]; then
     # search for files containing version markers
     sed -i.bak -e "/${VERSION_MARKER}/s/${CURRENT_VERSION}/${NEW_VERSION}/g" $FILES
-    sed -i.bak -e "/${VERSION_MARKER_NL}/n;s/${CURRENT_VERSION}/${NEW_VERSION}/g" $FILES
+    sed -i.bak -e "/${VERSION_MARKER_NL}/{n;s/${CURRENT_VERSION}/${NEW_VERSION}/g;}" $FILES
```

Comment:
I'm not sure if this is working.

I created a test file:

```
1.2.0-SNAPSHOT # MY_VERSION

some text

1.2.0-SNAPSHOT

some more text

# MY_VERSION
1.2.0-SNAPSHOT

blah

1.2.0-SNAPSHOT
# MY_VERSION
```

And I ran `$BROOKLYN_REPO/usage/scripts/change-version.sh MY 1.2.0-SNAPSHOT 1.2.0-CHANGED`.

The output file was:

```
1.2.0-CHANGED # MY_VERSION

some text

1.2.0-SNAPSHOT

some more text

# MY_VERSION
1.2.0-SNAPSHOT

blah

1.2.0-SNAPSHOT
# MY_VERSION
```

i.e. it didn't change it on the line after "# MY_VERSION".